charts/iter8lib/templates/_k-manifest-istio.tpl

```diff
@@ -0,0 +1,13 @@
+{{- define "k.manifest-istio" -}}
+{{ include "k.job" . }}
+---
+{{ include "k.spec.secret-istio" . }}
+---
+{{ include "k.spec.role" . }}
+---
+{{ include "k.spec.rolebinding" . }}
+---
+{{ include "k.result.role" . }}
+---
+{{ include "k.result.rolebinding" . }}
+{{- end }}
```

Comment:
I am a bit bothered by the istio specific items in `iter8lib`. I wonder if it makes more sense to create a `istiolib` or `iter8istio` that contains these. The dependency might be `istio-slo-validation` depends on `istiolib` depends on `iter8lib`.

Reply:
Should be updated to reflect new changes to `_k-manifest.tpl` (ie, conditionals and readiness).